server/server.go

```diff
@@ -470,12 +455,6 @@ func New(
 		logrus.Infof("unable to start config watcher: %v", err)
 	}
 
-	// Start a configuration watcher for the registries of the SystemContext
-	registriesPath := sysregistriesv2.ConfigPath(s.systemContext)
-	if _, err := s.StartConfigWatcher(registriesPath, s.ReloadRegistries); err != nil {
-		logrus.Infof("unable to start config watcher: %v", err)
-	}
-
```

Comment:
So this isn't needed any more?

Reply:
Yes, this is the main benefit of the change. Since the system context is now part of config we can reload the registries directly there. :) (see reload.go:35). The naming `ConfigWatcher` is a bit misleading because it indicates that we're watching a file for changes, which we don't. We just watch for SIGHUP.